sci-libs/dlib/dlib-19.15.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="Numerical and networking C++ library"
+DESCRIPTION="Dlib is a modern C++ toolkit containing machine learning \
+algorithms and tools for creating complex software in C++ to solve real \
+world problems."
+HOMEPAGE="http://dlib.net/"
+COPYRIGHT="2003-2018 Davis E. King"
+LICENSE="Boost v1.0"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/dclib/dlib-$portVersion.tar.bz2"
+CHECKSUM_SHA256="5340eeaaea7dd6d93d55e7a7d2fdb1f854a77b75f66049354db53671a202c11d"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	dlib1915$secondaryArchSuffix = $portVersion
+	lib:libdlib$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:liblapack$secondaryArchSuffix
+	lib:libopenblas$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	dlib1915${secondaryArchSuffix}_devel = $portVersion
+	devel:libdlib$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	dlib1915$secondaryArchSuffix == $portVersion base
+	"
+CONFLICTS_devel="
+	dlib${secondaryArchSuffix}_devel
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libgif$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:liblapack$secondaryArchSuffix
+	devel:libopenblas$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage dlib1915$secondaryArchSuffix \
+	$libDir/libdlib.so.$libVersion.0
+
+BUILD()
+{
+	mkdir -p build && cd "$_"
+
+	cmake .. \
+		-DBUILD_SHARED_LIBS=ON \
+		$cmakeDirArgs
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make install
+
+	mkdir -p $libDir/cmake/
+
+	prepareInstalledDevelLib libdlib
+
+	sed  -i "1i prefix=$prefix" \
+		$libDir/pkgconfig/dlib-1.pc
```

Comment:
I think we should fix `fixPkgconfig` to stop assuming that `prefix` is in the `.pc` file.